haproxy/dockerdir/usr/local/bin/readiness-check.sh

```diff
@@ -6,10 +6,12 @@ if [[ "${IS_PROXY_PROTOCOL}" = "yes" ]]; then
 fi
 
 MONITOR_USER='monitor'
+MONITOR_PASSWORD=$(cat /etc/mysql/mysql-users-secret/monitor)
 TIMEOUT=10
 MYSQL_CMDLINE="/usr/bin/timeout $TIMEOUT /usr/bin/mysql -nNE -u$MONITOR_USER"
 
-export MYSQL_PWD=${MONITOR_PASSWORD:-$PATH}
+export MYSQL_PWD=$MONITOR_PASSWORD
```

Comment:
```suggestion
export MYSQL_PWD=$(cat /etc/mysql/mysql-users-secret/monitor)
```

Reply:
removed in https://github.com/percona/percona-docker/pull/378/commits/ca9374116307d0241d2fa5e490f5d47f61d7a180